calendar-bundle/src/Resources/contao/dca/tl_calendar_events.php

```diff
@@ -910,7 +910,7 @@ public function getArticleAlias(DataContainer $dc)
 
 			while ($objAlias->next())
 			{
-				$arrAlias[$objAlias->parent][$objAlias->id] = $objAlias->title . ' (' . ($GLOBALS['TL_LANG']['COLS'][$objAlias->inColumn] ?: $objAlias->inColumn) . ', ID ' . $objAlias->id . ')';
+				$arrAlias[$objAlias->parent][$objAlias->id] = $objAlias->title . ' (' . (($GLOBALS['TL_LANG']['COLS'][$objAlias->inColumn] ?? null) ?: $objAlias->inColumn) . ', ID ' . $objAlias->id . ')';
```

Comment:
```suggestion
				$arrAlias[$objAlias->parent][$objAlias->id] = $objAlias->title . ' (' . ($GLOBALS['TL_LANG']['COLS'][$objAlias->inColumn] ?? $objAlias->inColumn) . ', ID ' . $objAlias->id . ')';
```

Shouldn‘t this suffice?